angular-workspace/projects/ni/nimble-angular/src/thirdparty/directives/router_link.ts

```diff
@@ -0,0 +1,404 @@
+/**
+ * [Nimble]
+ * Copied from https://github.com/angular/angular/blob/15.2.0/packages/router/src/directives/router_link.ts
+ * with the following modifications:
+ * - Hardcode `isAnchorElement` to `true` so that the directive will correctly set the `href` on elements within nimble that represent anchors
+ * - Make `href` a `@HostBindinding` to avoid using Angular's private sanitization APIs
```

Comment:
```suggestion
 * - Make `href` a `@HostBindinding` to avoid using Angular's private sanitization APIs. Implementations leveraging RouterLink should have a test ensuring sanitization is called.
```

Reply:
Seems like we are relying on Angular to do some magic on attributes named href? Is there anything to link to documenting that? If we can't find that can we say something explicitly about relying on that observed behavior?